README.md

```diff
@@ -1,8 +1,60 @@
-# MFSDA_Python
+# MFSDA: Multivariate Functional Shape Data Analysis
 
-This MFSDA_Python package is developed by Chao Huang and Hongtu Zhu from the [BIG-S2 lab](http://odin.mdacc.tmc.edu/bigs2/). 
+MFSDA_Python is a Python based package for statistical shape analysis.
 
-MFSDA_Python is a Python based package for statistical shape analysis. A multivariate varying coefficient model is introduced to build the association between the multivariate shape measurements and demographic information and other clinical variables. Statistical inference, i.e., hypothesis testing, is also included in this package, which can be used in investigating whether some covariates of interest are significantly associated with the shape information. The hypothesis testing results are further used in clustering based analysis, i.e., significant suregion detection. 
+A multivariate varying coefficient model is introduced to build the association between the multivariate shape measurements and demographic information and other clinical variables.
+
+Statistical inference, i.e., hypothesis testing, is also included in this package, which can be used in investigating whether some covariates of interest are significantly associated with the shape information.
+
+The hypothesis testing results are further used in clustering based analysis, i.e., significant suregion detection.
+
+## Modules
+
+### Loadable module
+
+The project a Slicer loadable allowing to XXX that internally leverage invokes command line modules described below.
+
+
+### CLI modules
+
+| CLI Name           | Description |
+|--------------------|-------------|
+| [MFSDA_createShapes][MFSDA_createShapes]             | ??? |
```

Comment:
@vicory 

Any suggestion for describing  `MFSDA_createShapes` and `MFSDA_run` ? 

Is `MFSDA_selectVariablesPCA`of any use ? I noticed it is not exposed a CLI like the previous two.

Cc: @juanprietob

Reply:
@vicory can you please let @jcfr know how to complete the description of each CLI?
After that has been addressed, it is all good by me, so @jcfr you can just merge without seeking further review.

Reply:
Would be nice to have a short description for each MFSDA CLIs :pray: